test-cases/gutenberg/unsupported-block-editing.md

```diff
@@ -24,27 +28,33 @@
 ##### TC002
 
 **Known Issues**
--  **[Android]** The update button is not greyed-out when changes are discarded
+-  **[Android-only]** The update button is not greyed-out when changes are discarded
 
-### Discarded edits are not persisted
+### Ensure discarded edits are not persisted
 
-1. Repeat steps 1 to 5 above
+1. Repeat steps 1 to 5 from TC001 above
 2. Edit the block content (e.g. update its text)
 3. Tap the Cancel button to discard changes and expect to be taken back to the block editor
 4. Expect the Update button to be greyed-out
-5. Inspect the HTML content of the post and expect it to **not** contain any of the changes
+5. Using HTML mode in the editor, view the HTML content of the post and expect it to **not** contain any of the changes
 
 ##### TC003
 
-### WP.com Business (Atomic) sites are supported
+### User can edit unsupported blocks on regular WP.com Business (Atomic) sites
 
-Repeat steps from TC001 using a post on a WP.com site with a Business Plan (i.e. Atomic site)
+Repeat TC001 and TC002 using a post on a WP.com site with a Business Plan (i.e. Atomic site)
 
 ##### TC004
 
 **Known Issues**
-This feature is disabled for Jetpack connected sites.
+- The Unsupported Block Editor is currently [not available on Jetpack connected self-hosted sites](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2471)
 
-### Self-hosted sites are supported
+### Self-hosted sites are supported (if they are not connected to WordPress via Jetpack)
```

Comment:
👋 @guarani , when logging in the Jetpack .org site using username and password, does it matter if Jetpack is connected or not? The testcase makes it look like "not connected" is supported, but unsure about when connected.

Can we add a new testcase to cover the connected case, or rephrase that bit in the current testcase (say, by removing the mention to connected or not)?

Reply:
That's a good point. Now we've disabled self-hosted sites I'm going to go ahead and remove the test case entirely. (I think self-hosted sites were supported when the user used their username/password to log in.)

Reply:
> I think self-hosted sites were supported when the user used their username/password to log in.

They were (before we found the issue with Classic editor popping up if Gutenberg is disabled) but I might be misunderstanding the testcase: when logging in with username/password the feature should actually have worked regardless of the Jetpack state (uninstalled, installed but not-connected, connected), right? 🤔